Documentation/security.md

```diff
@@ -21,8 +21,35 @@ Here are some security best practices:
 * To avoid the previous point: **share a full filesystem instead of just a directory in a filesystem if possible**. For example, a mounted partition or some file mounted with `mount -o loop`.
 * **Sharing devices from the host to the container is generally not recommended**. If you need to do it, you can find examples in the [block devices documentation](block-devices.md).
 
+## Incoming mounts
+
+Mounts can be added to a pod at runtime by using [`machinectl bind`][machinectl-bind] or when adding apps to a rkt pod with the [app experiment][app-experiment].
+
+This can be used in a malicious way although it requires privileges on the host.
+
+## File descriptors
+
+In certain situations, some file descriptors are passed between the host and the container which might have security implications.
+
+### Socket activation
+
+rkt supports [socket-activated services][rkt-socket-activated] and [socket-proxyd][socket-proxyd].
+This means some file descriptors will be passed from the host to the container when using those features.
+
+In particular, all file descriptors defined in the socket unit file accompanying the service file where rkt is started are passed to the container.
+
+### sd\_notify
+
+rkt supports the [sd_notify][sd_notify] mechanism to notify systemd on the host that the pod is ready.
+This means the container will receive a file descriptor which can be used to send arbitrary information to the systemd on the host.
```

Comment:
The process who has the fd is `systemd` in the pod and it is not controlled by the apps, so apps cannot send arbitrary information. Maybe you could just say this is used in `systemctl status...` to display whether the apps are initialized?